src/chrome/app/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "__MSG_appName__",
   "description": "__MSG_appDescription__",
-  "minimum_chrome_version": "38",
+  "minimum_chrome_version": "41.0.2272",
```

Comment:
So, this can be 41.0.2267:
https://github.com/uProxy/uproxy-networking/pull/174